src/shell/commands/cold_backup.cpp

```diff
@@ -157,15 +157,16 @@ bool ls_backup_policy(command_executor *e, shell_context *sc, arguments args)
 
 bool query_backup_policy(command_executor *e, shell_context *sc, arguments args)
 {
-    const std::string query_backup_policy_help = "<policy_name> [-b|--backup_info_cnt] [-j|--json]";
+    const std::string query_backup_policy_help =
+        "<-p|--policy_name> [-b|--backup_info_cnt] [-j|--json]";
     argh::parser cmd(args.argc, args.argv, argh::parser::PREFER_PARAM_FOR_UNREG_OPTION);
-    RETURN_FALSE_IF_NOT(cmd.pos_args().size() > 1,
+    RETURN_FALSE_IF_NOT(cmd.pos_args().size() <= 1,
```

Comment:
<-p|--policy_name> is required, but this judgement doesn't require this, `query_backup_policy -b xxx` is also considered as valid, right?